packages/modules/elasticsearch/src/index.ts

```diff
@@ -1 +1,5 @@
-export { ElasticsearchContainer, StartedElasticsearchContainer } from "./elasticsearch-container";
+export {
+  ELASTIC_SEARCH_HTTP_PORT,
```

Comment:
In Testcontainers for Java, those values are not exported. However, what we did lately is to document all the ports used by the containers implementations.

Reply:
I added this because it is convenient for me and it may be convenient for others. An alternative to exporting a constant is to copy and paste that constant into a project that uses `ElasticsearchContainer`.

By the way, thank you for your time for this review.

Reply:
Yep, totally make sense. I'm not saying this is a bad idea but just making sure we are aligned in the different language implementations :)